### PR TITLE
feat(pattern/textarea): create textarea pattern

### PR DESCRIPTION
**Pull request by gael-boyenval**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: https://design-system-adeo.atlassian.net/browse/MZC-52

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

File: src/pages/Components/TextArea/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<hintitem>
+  Always use a textarea to fill multi-line texts.
+</hintitem>
+
+<hintitem>
+  Allow users to resize the field height to see more or less the content.
+</hintitem>
+
+<hintitem>
+  A textarea should allow at least 3 lines of text without beinbg resized by the user
```
Comment (caqueste) [inline src/pages/Components/TextArea/index.md]: beinbg ?

Comment (gael-boyenval) [reply src/pages/Components/TextArea/index.md]: it's german

Comment (gael-boyenval) [reply src/pages/Components/TextArea/index.md]: yaaaaa

File: src/pages/Components/TextArea/specs.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+| State   | Property      | Token            |
+| ------- | ------------- | ---------------- |
+| **all** | border-radius | `radius.m` (4px) |
+| **all** | border-width  | `border.s` (1px) |
+| **all** | padding       | `1mu` (16px)     |
+
+### CSS Tokens : state colors
+
+**You must always use the tokens, the colors are only there for your information.**
```
Comment (tiagoux) [inline src/pages/Components/TextArea/specs.md]: You must always use the tokens, the colors are listed here for information purposes only.

**Pull request merged**